ClientCore/CnCNet5/GameCollection.cs

```diff
@@ -197,7 +197,7 @@ public void Initialize()
             };
 
             // CnCNet chat.
-            CnCNetGame[] otherGames =
+            CnCNetGame[] otherGames = 
```

Comment:
unneeded

Reply:
> unneeded

wdym

Reply:
the change is just adding a trailing whitespace, random unwanted change

Reply:
> > unneeded
> 
> wdym

Adding a white space after `=` is unneeded, indicating to a bad code formatting. This space character was introduced by someone else, but in order to merge it into develop, such an unnecessary change should be manually reverted.

Reply:
Done by 17437662c37e4b55f0d2ea67b0823184eaf1dae2.